flink-java/src/main/java/org/apache/flink/api/java/io/CsvInputFormat.java

```diff
@@ -64,15 +65,24 @@
 	private transient int commentCount;
 
 	private transient int invalidLineCount;
+
+	private PojoTypeInfo<OUT> pojoTypeInfo = null;
 	
 	
 	public CsvInputFormat(Path filePath) {
 		super(filePath);
-	}	
+	}
+
+	public CsvInputFormat(Path filePath, PojoTypeInfo<OUT> pojoTypeInfo) {
+		super(filePath);
+
+		Preconditions.checkNotNull(pojoTypeInfo, "Type information must be required to set values to pojo");
```

Comment:
The error message reads a bit weird. How about "The TypeInformation is required for getting the POJO fields."